src/main/resources/application-SENDER.properties

```diff
@@ -32,6 +32,16 @@ application.keyStoreName=
 application.keyStorePassword=
 application.keystoreAliasName=
 application.connectorUUID=
+
+application.trustStoreName=
+application.trustStorePassword=
+
+#define if the connector is used as receivere
+application.isReceiver=false
+
+#IDSCP2 enabled
+application.idscp2.isEnabled=true
```

Comment:
Can we set this one by default to false?
Let's start with rest examples first :)